oak/proto/oak_api.proto

```diff
@@ -41,4 +41,6 @@ enum ChannelHandle {
   LOGGING = 1;   // Write half of channel
   GRPC_IN = 2;   // Read half of channel
   GRPC_OUT = 3;  // Write half of channel
-}
\ No newline at end of file
+  STORAGE_READ = 4;
+  STORAGE_WRITE = 5;
```

Comment:
Update README.md to describe the new well-known channel handles?

Reply:
Done.